### ISSUE TITLE
Test builds do not use installation method indicated in README.md

### ISSUE THREAD

**Issue by mjwoudstra** [QuTech-Delft/qtt#617]
The test builds that run to validate pull requests use an installation command (`pip install -r requirements.txt`) which is different from the installation command in the README.md file (`pip install -e .`), which is what end users would use. 
There are 2 problems with this:
1. It violates the DRY principle (setup.py and requirements.txt)
2. The test build does not test the installation procedure that an end user would use.

It there a reason to have 2 different install methods?

For example, issue #616  was not caught in the test build for pull request "[OP-61] Improve qtt installation." (to solve issue #613 ). In that particular case, the user install method does not include the PyQt5 dependency (in setup.py), whereas the test build install method _does_ include it (in requirements.txt).

[OP-61]: https://qutech-sd.atlassian.net/browse/OP-61

**Comment by CumulonimbusCalvus** [comment]
Yes, I completely agree.

This was in the past not possible since the PiPy version of qcodes was not updated for a long time. It meant we had to install the package directly from github, which is only possible using the `requirements.txt`. The same holds for `qilib`, which is being resolved in this sprint (DEM-790) by the SE-team.

**Comment by mjwoudstra** [comment]
I see. So in that case I suggest we change the README.md to use 
`pip install -r requirements.txt -e .`
as @peendebak had already suggested in issue #611 .

We still have the duplicate dependencies list in setup.py though. Not sure if we can remove all the dependencies there, but that's lower priority.

**Comment by CumulonimbusCalvus** [comment]
The only problem with the `pip install -r requirements.txt -e .` solution is that once we release on PyPi the installation does not work anymore, since you cannot use any `requirements.txt` with PyPi.

I hope that once the SE-Team sprint is done, we have qilib on PyPi and move all dependencies to the `setup.py` and only have a `requirements.txt` or `developement_requirements.txt` for building the sphinx documentation.

**Comment by peendebak** [comment]
qilib is on pypi, we use the pip install now
